### ISSUE TITLE
intSyntax.great_tm

### ISSUE THREAD

**Issue by xrchz** [HOL-Theorem-Prover/HOL#629]
`intSyntax.great_tm` is not a great name (no pun intended), and is inconsistent with `numSyntax.greater_tm`. This issue is to use `greater` in `intSyntax`.

**Comment by arolle** [comment]
A fix seem to be
```bash
grep -lZr --include \*.sig --include \*.sml  "great_\|_great\>" "$HOLDIR" \
  | xargs -0 -I% sed -i 's/_great\>/_greater/g; s/great_/greater_/g' %
```
which implies the following changes (see #823):
```
intSyntax.great_tm   -> intSyntax.greater_tm
intSyntax.dest_great -> intSyntax.dest_greater
intSyntax.mk_great   -> intSyntax.mk_greater
                                                                            
int_arithTheory.add_to_great -> int_arithTheory.add_to_greater
                                                                            
realSyntax.great_tm   -> realSyntax.greater_tm
realSyntax.dest_great -> realSyntax.dest_greater
realSyntax.mk_great   -> realSyntax.mk_greater
```

**Comment by mn200** [comment]
Closed by #823
